### PR TITLE
Change timestop invocation

### PR DESCRIPTION
**Pull request by Shadowtail117**
## About The Pull Request

Changes the invocation (the shout) for timestop from "TOKI YO TOMARE" to "TOKI WO TOMARE".

## Why It's Good For The Game

This spell is based on DIO, obviously, and he says it with a W, not a Y. Plus, using "yo" in this way isn't even proper Japanese.

## Changelog
:cl:
spellcheck: Fixed timestop invocation
/:cl:

### CODE REVIEW

**Comment by ATH1909** [comment]
https://github.com/tgstation/tgstation/pull/45585

The invocation is intentionally spelled that way.

**Comment by Shadowtail117** [comment]
The Japanese particle 「よ」"yo" is used at the end of a sentence to (at the risk of oversimplifying) indicate that the sentence was to explain something. For example, 「これは水ですよ」"kore wa mizu desu yo" would translate to "This is water(, you know)." Putting it in the middle of a sentence, especially between the object and verb, makes no sense grammatically.

In contrast, the particle 「を」"wo" is used as an "object" marker. Japanese follows a Subject-Object-Verb sentence structure, in contrast with English's Subject-Verb-Object sentence structure. To help clarify where the subject ends and the object begins, or when the object ends and the verb begins, a subject marker「は」"wa" and object marker「を」"wo" are used. 「時をとまれ」"toki wo tomare" means "stop time", and the object marker here designates 「時」"toki; time" as the object. So, this makes perfect grammatical sense.

tl;dr "toki yo tomare" makes no sense grammatically. It should be, in all accounts, "toki wo tomare".

**Comment by remove32** [comment]
image
image
image
Dio is speaking to Time and commanding it to stop

**Comment by gonenoculer5** [comment]
as someone who vaguely studied japanese in high school, @remove32 is correct, Yo is for emphasizing something/commanding something/someone in an aggressive form, in this case DIO commanding that time halt by his order

**Comment by TheSwain** [comment]
so should i close this? i guess we can close this
